meson.build

```diff
@@ -135,8 +135,8 @@ endif
 subdir('va')
 subdir('pkgconfig')
 
-doxygen = find_program('doxygen', required: false)
+doxygen = find_program('doxygen', required : get_option('enable_docs') == true)
```

Comment:
If enable_docs is true, then this evaluates to `true == true`, which in turn evaluates to true

If enable_docs is false, then this evaluates to `false == true`, which in turn evaluates to false.

You can just use the option directly. :)